docs/platforms/javascript/common/user-feedback/index.mdx

```diff
@@ -28,16 +28,22 @@ The embeddable JavaScript widget allows users to submit feedback from anywhere i
 
 To set up the integration, add the following to your Sentry initialization. There are many options you can pass to the integration constructor. See the [configuration documentation](/platforms/javascript/user-feedback/configuration/) for more details.
 
-### Session Replay
-
-The User Feedback Widget integrates easily with <PlatformLink to="/session-replay/">Session Replay</PlatformLink>. First, make sure that the Session Replay integration is configured correctly and that `replaysOnErrorSampleRate` is greater than 0. When this is done, the Replay SDK will buffer up to 30 seconds worth of events until the user *opens* the User Feedback Widget. If the user submits feedback, you'll be able to view the Feedback (including the replay), inside of Sentry's web application.
-
 <SignInNote />
 
 <PlatformContent includePath="user-feedback/setup" />
 
 By default, this will insert the widget into the bottom right corner of your website. You're free to customize nearly every aspect of the widget, including replacing it completely with your own UI.
 
+On SDK version 8.0.0 and above, users can send screenshots with their feedback. If you're using on-prem, you also need release 24.4.2 and above. You can configure this using the `showScreenshot` option, by default it is set to `true`. Screenshots aren't supported on mobile devices, so the screenshot button will be hidden automatically in this case.
+
+<Alert level="">
+  Screenshots use your [attachments quota](/product/accounts/quotas/manage-attachments-quota). All plans come with 1GB of attachments, which is approxiamately 2500 screenshots.
+</Alert>
+
+### Session Replay
+
+The User Feedback widget integrates easily with <PlatformLink to="/session-replay/">Session Replay</PlatformLink>. First, make sure that the Session Replay integration is configured correctly and that `replaysOnErrorSampleRate` is greater than 0. When this is done, the Replay SDK will buffer up to 30 seconds of the user's session until the user *opens* the User Feedback widget. If the user submits feedback, you'll be able to view the feedback (including the replay), inside of Sentry's web application.
```

Comment:
>  inside of Sentry's web application.

@lizokm should we say "inside Sentry" ? I wonder if we have a consistent way to address this across the docs?

Reply:
I think we can just say "in Sentry.io"